docs/src/main/sphinx/sql/alter-materialized-view.rst

```diff
@@ -19,6 +20,22 @@ materialized view does not exist. The error is not suppressed if the
 materialized view does not exist, but a table or view with the given name
 exists.
 
+.. _alter-materialized-view-set-properties:
+
+SET PROPERTIES
+^^^^^^^^^^^^^^
+
+The ``ALTER MATERALIZED VIEW SET PROPERTIES``  statement followed by some number
+of ``property_name`` and ``expression`` pairs applies the specified properties
+and values to a materialized view. Ommitting an already-set property from this
+statement leaves that property unchanged in the materialized view.
+
+A property in a ``SET PROPERTIES`` statement can be set to ``DEFAULT``, which
+reverts its value back to the default in that materialized view.
```

Comment:
Do defaults exist for every property @sunningCGo ? What happens if there is a null default but nulls are not allowed for a property?

Reply:
@hashhar there are tests for this behavior in `TestSetPropertiesTask.java`.